activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -2901,46 +2923,111 @@ public boolean isShutdownOnSlaveFailure() {
     }
 
     /**
+     * This value shutdownOnSlaveFailure is deprecated and will be removed in a future release. Use shutdownOnStandbyFailure instead.
+     * @param shutdownOnSlaveFailure
      * @org.apache.xbean.Property propertyEditor="org.apache.activemq.util.BooleanEditor"
      */
     public void setShutdownOnSlaveFailure(boolean shutdownOnSlaveFailure) {
         this.shutdownOnSlaveFailure = shutdownOnSlaveFailure;
+        this.shutdownOnStandbyFailure = shutdownOnSlaveFailure;
+        LOG.warn("shutdownOnSlaveFailure is deprecated and will be removed in a future release. Use shutdownOnStandbyFailure instead.");
+    }
+
+    public boolean isShutdownOnStandbyFailure() {
+        return shutdownOnStandbyFailure;
+    }
+
+    /**
+     * @org.apache.xbean.Property propertyEditor="org.apache.activemq.util.BooleanEditor"
+     */
+    public void setShutdownOnStandbyFailure(boolean shutdownOnStandbyFailure) {
+        this.shutdownOnStandbyFailure = shutdownOnStandbyFailure;
     }
 
     public boolean isWaitForSlave() {
         return waitForSlave;
     }
 
     /**
+     * This value waitForSlave is deprecated and will be removed in a future release. Use waitForStandby instead.
+     * @param waitForSlave
      * @org.apache.xbean.Property propertyEditor="org.apache.activemq.util.BooleanEditor"
      */
     public void setWaitForSlave(boolean waitForSlave) {
         this.waitForSlave = waitForSlave;
+        this.waitForStandby = waitForSlave;
+        LOG.warn("waitForSlave is deprecated and will be removed in a future release. Use waitForStandby instead.");
     }
 
     public long getWaitForSlaveTimeout() {
         return this.waitForSlaveTimeout;
     }
 
+    /**
+     * This value waitForSlaveTimeout is deprecated and will be removed in a future release. Use waitForStandbyTimeout instead.
+     * @param waitForSlaveTimeout
+     */
     public void setWaitForSlaveTimeout(long waitForSlaveTimeout) {
         this.waitForSlaveTimeout = waitForSlaveTimeout;
+        this.waitForSlaveTimeout = waitForStandbyTimeout;
+        LOG.warn("waitForSlaveTimeout is deprecated and will be removed in a future release. Use waitForStandbyTimeout instead.");
+
+    }
+
+    public boolean isWaitForStandby() {
+        return waitForStandby;
+    }
+
+    /**
+     * @org.apache.xbean.Property propertyEditor="org.apache.activemq.util.BooleanEditor"
+     */
+    public void setWaitForStandby(boolean waitForStandby) {
+        this.waitForStandby = waitForStandby;
+    }
+
+    public long getWaitForStandbyTimeout() {
+        return this.waitForStandbyTimeout;
+    }
+
+    public void setWaitForStandbyTimeout(long waitForStandbyTimeout) {
+        this.waitForStandbyTimeout = waitForStandbyTimeout;
     }
 
     /**
-     * Get the passiveSlave
+     * This value passiveSlave is deprecated and will be removed in a future release. Use standbyInstance instead.
+     * @deprecated
```

Comment:
`@deprecated` is good, I'd suggest for all the methods, do it full java style - i.e.

javadocs have `@deprecated - AMQ-7514 in the move to inclusive nomenclature` and method signatures have the `@Deprecated` annotation.